bsip-0076.md

```diff
@@ -0,0 +1,88 @@
+    BSIP: 0076
+    Title: Threshold for price feeds through voting
+    Authors: Abit More
+    Status: Draft
+    Type: Informational
+    Created: 2019-09-25
+
+# Abstract
+
+This BSIP defines a process to influence the pegging status of committee-owned smartcoins through poll voting.
+
+# Motivation
+
+In order to maintain the peg of the smartcoins, smartcoins holders now have the right to be always
+possible to buy collateral at a "fair" price. For committee-owned smartcoins, the collateral is BTS.
+However, when the BTS token is being dumped in Centralized Exchanges with fake volume and even fake supply,
+the value of the token is distorted, thus, forcing BTS token holders who have their BTS token trapped in
+debt positions as collateral to sell their collateral at the distorted price via margin calls or
+force-settlements is unfair.
+The same situation applies to non-committee-owned smart coins as well.
+
+Smartcoins are products of the BitShares platform, their rules are defined by the platform
+for the benefit of the platform and the BTS token holders.
+If a product brings more harm than benefit to the platform, its rules need to be optimized,
+under extreme conditions, the product needs to be halted.
+For smartcoins, when the cost to maintain the peg is too high to afford, it's time to give up the peg.
+
+# Rationale
+
+Ideally a smartcoin owner should be able to define a set of conditions about when to give up the peg
+and how to de-peg, but it needs time to design and implement. Before the final solution got worked out,
+it's up to the price feed producers to execute the will for the asset owner on whether to maintain
+the peg at a given point and how to de-peg. For committee-owned smartcoins, the owner is actually
+all BTS holders who could express their will by voting, the price feed producers are the active witnesses.
+
+# Specification
+
+The BTS token holders define a threshold, if the market trading price of BTS is below the threshold,
+witnesses should feed the threshold but not the market trading price, thus would effectively break
+the peg as intended. When the market trading price of BTS is above the threshold, witnesses should
+feed the market trading price, so the peg will be restored automatically. The mechanism is similar to
```

Comment:
no price feed script changes to automatically switch over from fake price feeds to real feeds once the BTS price exceeds the set threshold? This probably won't be automatic since it's not been coded in such a manner